tests/bench_runtime_coreapis/tests/01-run.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2018 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import sys
+
+
+def testfunc(child):
+    child.expect_exact('[SUCCESS]')
+
+
+if __name__ == "__main__":
+    sys.path.append(os.path.join(os.environ['RIOTTOOLS'], 'testrunner'))
+    from testrunner import run
+    sys.exit(run(testfunc))
```

Comment:
According to PEP8 guidelines there should be a newline at the end of the file. Do we enforce PEP8?

Reply:
I think so, will fix

Reply:
there is a newline at the end of the file, and `flake8` is also not complaining. Do you think there needs to be an additional newline?

Reply:
I think it's fine